packages/forklift-console-plugin/src/modules/Plans/views/create/components/PlanNameTextField.tsx

```diff
@@ -23,16 +23,27 @@ export const PlanNameTextField: React.FC<PlanNameTextFieldProps> = ({
   const { t } = useForkliftTranslation();
   const [isUpdated, setIsUpdated] = React.useState(false);
 
+  const getInvalidHelperText = () => {
```

Comment:
why in tsx file ?

Reply:
To avoid sending 3 params: value, validated and, t func